bazel/api-gen/rendering/templates/class-method-info.tsx

```diff
@@ -37,7 +37,6 @@ export function ClassMethodInfo(props: {
       ) : (
         <div>
           <DeprecatedLabel entry={entry} />
-          <span>{entry.deprecationMessage}</span>
```

Comment:
Maybe I am misunderstanding, but are we not including the deprecation message anymore?

Reply:
This one was duplicated, it's handled by the `DeprecatedLabel `.